lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
+    }
+
+    const { delimiterChars } = this.props
+
+    e.preventDefault()
+
+    // get the text data from the clipboard
+    const data = e.clipboardData.getData('Text')
+
+    if (data && delimiterChars.length > 0) {
```

Comment:
If there are no delimiterChars, then ReactTags can't do its job, I think? It needs a list to be able to chunk up input in the first place so if it does not have that list it can't convert string data into tags. Some options are to issue a warning ("no delimiter characters specified for onPaste handling by ReactTags") or to simply break out of the handler function before any code actually gets run.

@i-like-robots is the idea that delimiterChars and delimiters are mutually exclusive (e.g. if you need real letters, don't use numbers, and vice versa) or are they complementary? (in which case I might try to build a map that can tell which key is which keyboard event code, because the difference between those two is still ridiculous =D)